lib/titiler-pgstac-api/runtime/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION
+
+FROM --platform=linux/amd64 public.ecr.aws/lambda/python:${PYTHON_VERSION}
+
+WORKDIR /tmp
+RUN python -m pip install pip -U
+
+COPY runtime/requirements.txt requirements.txt
+RUN python -m pip install -r requirements.txt "mangum>=0.14,<0.15" -t /asset
+
+# Reduce package size and remove useless files
+RUN cd /asset && find . -type f -name '*.pyc' | while read f; do n=$(echo $f | sed 's/__pycache__\///' | sed 's/.cpython-[0-9]*//'); cp $f $n; done;
```

Comment:
This line kind of blows my mind. So, we're finding all `*.pyc` files, reading through them line by line, then copying the `.pyc` file to the path mentioned within the file after removing `__pycache__` and/or `.cpython*` from the path?

What does this do?

Reply:
@alukach 

> So, we're finding all *.pyc files, reading through them line by line, then copying the .pyc file to the path mentioned within the file after removing __pycache__ and/or .cpython*

Yep, we get rid of the non compiled `.py` files that are useless because we have the `.pyc` files. This wins maybe 10ms when we start the handler 😄

Reply:
This is just a small optimization I do on all my lambdas

Reply:
So do these compiled files get created from the `pip install` command? Is there any value to running `python -m compileall .` before this?

Btw, doing some research on this, I found [this SO answer](https://stackoverflow.com/a/53079956/728583) which linked to [this article] (https://blog.mapbox.com/aws-lambda-python-magic-e0f6a407ffc6). I was about to link it until I realized that **you wrote the article.** 😂

Reply:
> So do these compiled files get created from the pip install command? Is there any value to running python -m compileall . before this?

Yes they are created on `pip install` command

Reply:
> Btw, doing some research on this, I found [this SO answer](https://stackoverflow.com/a/53079956/728583) which linked to [this article](https://blog.mapbox.com/aws-lambda-python-magic-e0f6a407ffc6). I was about to link it until I realized that you wrote the article. 😂

Old days ;-)